Cargo.toml

```diff
@@ -3,6 +3,9 @@ name = "pgcat"
 version = "1.1.1"
 edition = "2021"
 
+[profile.release]
+debug = true
```

Comment:
The executable is gonna be quite large. Not an issue for production imo.

Reply:
I committed it unnecessarily

Reply:
It's great for debugging production issues, otherwise the backtraces are pretty slim.